modules/log-collector-resources-scheduling.adoc

```diff
@@ -6,51 +6,85 @@
 [id="log-collector-resources-scheduling_{context}"]
 = Configuring resources and scheduling for logging collectors
 
-Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
-
-The applicable stanzas for the `ClusterLogging` CR when using multiple log forwarders in a deployment are `managementState` and `collection`. All other stanzas are ignored.
+Administrators can modify the resources or scheduling of the collector by configuring the `collector` field in a `ClusterLogForwarder` custom resource (CR).
 
 .Prerequisites
 
 * You have administrator permissions.
-* You have installed the {clo} version 5.8 or newer.
-* You have created a `ClusterLogForwarder` CR.
+* You have installed {clo}.
+* You have created a a `ClusterLogForwarder` CR.
 
 .Procedure
 
-. Create a `ClusterLogging` CR that supports your existing `ClusterLogForwarder` CR:
+. Update the `ClusterLogForwarder` CR:
 +
-.Example `ClusterLogging` CR YAML
+.Example `ClusterLogForwarder` CR YAML
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
-kind: ClusterLogging
+apiVersion:  observability.openshift.io/v1
+kind: ClusterLogForwarder
 metadata:
-  name:  <name> # <1>
-  namespace: <namespace> # <2>
+  name:  <name>
+  namespace:
+    name: <namespace> 
```

Comment:
This should actually just be `namespace: <namespace>`. The way it was previously written was correct.

Reply:
done